src/test/java/com/redhat/cloud/notifications/OpenApi2Test.java

```diff
@@ -0,0 +1,68 @@
+package com.redhat.cloud.notifications;
+
+import com.reprezen.kaizen.oasparser.OpenApi3Parser;
+import com.reprezen.kaizen.oasparser.model3.OpenApi3;
+import com.reprezen.kaizen.oasparser.val.ValidationResults;
+import io.quarkus.test.common.http.TestHTTPResource;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.Assert;
```

Comment:
This should be ``import static org.junit.jupiter.api.Assertions.fail;`` now it's importing the JUnit4-legacy-API.